src/components/templates/PartyMap/PartyMapRouter.tsx

```diff
@@ -1,18 +0,0 @@
-import React, { FC } from "react";
-import { useRouteMatch, Switch, Route } from "react-router-dom";
-
-import useConnectCurrentVenue from "hooks/useConnectCurrentVenue";
-
-import PartyMap from "./PartyMap";
-
-export const PartyMapRouter: FC = () => {
-  const match = useRouteMatch();
-  useConnectCurrentVenue();
-
-  return (
-    <Switch>
-      <Route path={`${match.path}/:roomTitle`} component={PartyMap} />
```

Comment:
The detail I added on the original comment asking if we could remove this `PartyMapRouter` component said that this route is still required, for nested sub-venues on a PartyMap (eg. jazz bar), and explained some nuances around concerns it would raise if it was added to the main app router for venues/etc incapable of displaying sub-venues.

Were these taken into account before this got removed?

Reply:
see https://github.com/sparkletown/sparkle/pull/1105#discussion_r565938578